Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu
```

Comment:
Copy `Cirq/examples` into this container so users can execute commands like `docker run -it cirq/cirq python3 bell_inequality.py.` Big nit: making `python` the default will look better. Currently,  `docker run -it cirq/cirq python bell_inequality.py` doesn't work because python2 is used instead of python3 (by using `FROM python:3` instead).

To copy:
```
ADD examples/*.py /examples/
WORKDIR /examples
```